Doc/library/sqlite3.rst

```diff
@@ -2269,8 +2339,67 @@ Explanation
 Transaction control
 ^^^^^^^^^^^^^^^^^^^
 
-The :mod:`!sqlite3` module does not adhere to the transaction handling recommended
-by :pep:`249`.
+.. _sqlite3-transaction-control-autocommit:
```

Comment:
While from a greenfield perspective it would be best to use `transaction-control` rather than `controlling-transactions` for this, the latter is what's used for the ref target label of the section this belongs to, which why my previous (unused) suggestions used them. it seems inconsistent, potentially confusing and breaks the logical hierarchy to arbitrarily use a different spelling in the subsections.

Therefore, I suggest either:

* Changing them all back to what I recommended, i.e. `sqlite3-controlling-transacations-autocommit`, or (much less work)
* Just adding another ref target label, `sqlite3-transaction-control`, to the `Transaction control` section.

Reply:
I prefer the latter.